### PR TITLE
[Feature] support ava-kinetics

### PR DESCRIPTION
**Pull request by hukkai**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily got feedback.
If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduces changes that break the back-compatibility of this repo?
If so, please describe how it breaks the compatibility and how users should modify their codes to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools should be used to fix the potential lint issues.
2. The modification should be covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation should be modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: tools/data/ava_kinetics/README.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+```shell
+python3 extract_rgb_frames.py --avakinetics_root=$AVAKINETICS_ROOT \
+                              [--num_workers=$NUM_WORKERS ]
+```
+
+Arguments:
+
+- `avakinetics_root`: the directory to save the ava-kinetics dataset. Defaults to `$MMACTION2/data/ava_kinetics`.
+- `num_workers`: number of workers used to cut videos. Defaults to -1 and use all available cpus.
```
Comment (cir7) [inline tools/data/ava_kinetics/README.md:96]:
```suggestion
- `num_workers`: number of workers used to extract frames. Defaults to -1 and use all available cpus.
```

File: tools/data/ava_kinetics/README.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+python3 prepare_annotation.py --avakinetics_anotation=$AVAKINETICS_ANOTATION \
+                              --avakinetics_root=$AVAKINETICS_ROOT \
+                              [--num_workers=$NUM_WORKERS]
+```
+
+Arguments:
+
+- `avakinetics_anotation`: the directory to ava-kinetics anotations. Defaults to `./ava_kinetics_v1_0`.
+- `avakinetics_root`: the directory to save the ava-kinetics dataset. Defaults to `$MMACTION2/data/ava_kinetics`.
+- `num_workers`: number of workers used to cut videos. Defaults to -1 and use all available cpus.
```
Comment (cir7) [inline tools/data/ava_kinetics/README.md:114]:
```suggestion
- `num_workers`: number of workers used to prepare annotation. Defaults to -1 and use all available cpus.
```

File: tools/data/ava_kinetics/prepare_annotation.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        if line[0] not in lookup:
+            continue
+        flag = False
+        for start, end, video_path in lookup[line[0]]:
+            if start < float(line[1]) < end:
+                flag = True
+                break
+        if flag is False:
+            continue
+        string = '%s,%d,%.3f,%.3f,%.3f,%.3f,%d,-1\n' % (
```
Comment (ly015) [inline tools/data/ava_kinetics/prepare_annotation.py:46]: As a convention, we usually use f-string in OpenMMLab codebases.

File: tools/data/ava_kinetics/extract_rgb_frames.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```
Comment (ly015) [inline tools/data/ava_kinetics/extract_rgb_frames.py:1]: Can we reuse the script `tools/data/build_rawframes.py` like: https://github.com/open-mmlab/mmaction2/blob/dev-1.x/tools/data/kinetics/extract_rgb_frames.sh?

Comment (hukkai) [reply tools/data/ava_kinetics/extract_rgb_frames.py:1]: Yes we can. But I did not try that and I am not sure about the the correctness of using this script.

**Review by cir7 — APPROVED** [review]

**Pull request merged**